dashboard/lib/services/script_seed.rb

```diff
@@ -247,7 +247,21 @@ def self.import_script(script_data)
       # Needed because we already have some Scripts with invalid names
       script_to_import.skip_name_format_validation = true
       Script.import! [script_to_import], on_duplicate_key_update: get_columns(Script)
-      Script.find_by!(name: script_to_import.name)
+
+      # activerecord-import doesn't trigger callbacks for imported models, and
+      # Scripts rely on the after_save hook to invoke `generate_plc_objects`,
+      # so we invoke it manually.
+      #
+      # see https://github.com/zdennis/activerecord-import#callbacks
+      #
+      # Note that we use activerecord-import extensively in the script seeding
+      # process, so we may end up needing to manually invoke these callbacks
+      # for more models than just Script, in which case we should probably
+      # reassess the pattern being used here.
+      imported_script = Script.find_by!(name: script_to_import.name)
+      imported_script.run_callbacks(:save)
+      imported_script.run_callbacks(:create)
```

Comment:
Do we want to run the `create` callbacks even if the script wasn't created?

Reply:
oh, probably not! I'll add some logic as a follow-up to detect when we're in the "create" vs "update" case